README.rst

```diff
@@ -158,7 +158,7 @@ context where the documents it validates cannot be malformed (because
 they are represented as DOM trees). So salve contains no functionality
 to handle problems with well-formedness. Salve **can be used on
 malformed documents**, provided you take care of reporting
-malformedness issues yourself and strategize how you will pass events
```

Comment:
"malformedness" is actually the correct word here. It is a term of art among people working with XML and its related technologies. It's opposite, "well-formedness", is more common.